runtime-modules/content/src/benchmarks/mod.rs

```diff
@@ -1554,23 +1538,23 @@ where
             starts_at: Some(System::<T>::block_number() + T::BlockNumber::one()),
             whitelist: whitelisted_members.into_iter().map(|(_, id)| id).collect(),
         }),
-    )?;
+    ).unwrap();
 
-    Ok((video_id, participant_id, participant_account_id))
+    Ok((nft_data, participant_id, participant_account_id))
 }
 
 fn setup_nft_in_open_auction<T>(
     account_id: T::AccountId,
     actor: ContentActor<T::CuratorGroupId, T::CuratorId, T::MemberId>,
     video_id: T::VideoId,
-    non_channel_owner: Option<T::MemberId>,
-) -> Result<(T::VideoId, T::MemberId, T::AccountId), DispatchError>
+    non_channel_owner: bool,
+) -> Result<(NftData<T>, T::MemberId, T::AccountId), DispatchError>
 where
     T::AccountId: CreateAccountId,
     T: RuntimeConfig,
 {
     let whitelist_size = Pallet::<T>::max_auction_whitelist_length();
-    let whitelisted_members = (0..(whitelist_size as usize))
+    let whitelisted_members = (1..(whitelist_size as usize))
```

Comment:
```suggestion
    let whitelisted_members = (1..=(whitelist_size as usize))
```
(otherwise the whitelist will always be 1 member short)

Reply:
Same issue as in the previous comment (https://github.com/Joystream/joystream/pull/4184#discussion_r960412177)